x/oracle/client/tests/suite.go

```diff
@@ -95,3 +96,18 @@ func (s *IntegrationTestSuite) TestDelegateFeedConsent() {
 		})
 	}
 }
+
+func (s *IntegrationTestSuite) TestQueryFeedDelegate() {
+	val := s.network.Validators[0]
```

Comment:
This should be table-driven (like other tests) where we have three cases:

1. A valid request
2. An invalid request (a bogus address)
3. A valid request but with a non-existent validator